src/Npgsql/NpgsqlConnection.cs

```diff
@@ -1584,6 +1584,54 @@ public override DataTable GetSchema()
         public override DataTable GetSchema(string? collectionName, string?[]? restrictions)
             => NpgsqlSchema.GetSchema(this, collectionName, restrictions);
 
+        /// <summary>
+        /// Asynchronously returns the supported collections
+        /// </summary>
+        /// <param name="cancellationToken">The token to monitor for cancellation requests. The default value is None.</param>
+        /// <returns>The collection specified.</returns>
+#if NET // i.e. >= NET5_0
```

Comment:
This is new to me, thanks! Can you submit another PR for the places in the codebase where we conditional-compile for .NET 5?

Reply:
I will do it 👍